packages/core/src/Stores/traders-hub-store.js

```diff
@@ -868,4 +845,19 @@ export default class TradersHubStore extends BaseStore {
     setIsSetupRealAccountOrGoToDemoModalVisible(value) {
         this.is_setup_real_account_or_go_to_demo_modal_visible = value;
     }
+
+    setTradingPlatformAvailability() {
+        this.root_store.client.trading_platform_available_accounts.forEach(account => {
+            if (account.product === 'synthetic') {
```

Comment:
We need to talk to the BE team to see why they are sending synthetic as product name instead of standard. This was recently changed to Standard.